tests/test_jmcomic/test_jm_async_feature.py

```
@@ -0,0 +1,102 @@
+        async_count = 0
+
+        class AsyncCounter(Feature):
+            def invoke(self, option, **kwargs):
+                nonlocal async_count
+                async_count += 1
+
+        asyncio.run(download_album_async(album_id, self.option, extra=AsyncCounter()))
+
+        # 核心断言：触发次数一致
+        self.assert_sync_async_equal(sync_count, async_count, 'feature.invoke_count (album)')
+        self.assertGreater(sync_count, 0, 'album(438516) 有 1 章，应至少触发 1 次')
```

Ruff (0.15.15)

[warning] String contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF001)